### PR TITLE
📖 on consent update request documentation 

### PR DESCRIPTION
**Pull request by zhouyx**
Merge when 1530320824316 hit prod

### CODE REVIEW

File: extensions/amp-consent/amp-consent.md [hunk 1]
```diff
@@ -144,6 +144,19 @@ current user to the 3rd party vendor extensions.
 
 Unlike consent state, this `shareData` is not persisted in client side storage.
 
+#### onUpdateHref
+`onUpdateHref`: Instructs AMP to make a CORS POST request with credentials to the specified URL whenever the stored consent state changes.
```
Comment (lannka) [inline extensions/amp-consent/amp-consent.md]: https only?

File: extensions/amp-consent/amp-consent.md [hunk 2]
```diff
@@ -388,77 +401,77 @@ Example:
...
 
 Use [`<amp-geo>`](https://www.ampproject.org/docs/reference/components/amp-geo) to configure a country group and hook up the country group to `promptIfUnknownForGeoGroup` attribute in `<amp-consent>`. If the user accesses the AMP page from a country that's configured in the list, the appropriate consent UI will be invoked.
 
 ##### Can I make the consent UI blocking?
 
-Yes. The UI is not prescriptive. If you do end up providing a non-blocking version of the UI, dismissing the UI will lead to a `dismiss` state. It is up to each vendor (ads & analytics) how they would process `accept`, `reject` and `dismiss`. 
+Yes. The UI is not prescriptive. If you do end up providing a non-blocking version of the UI, dismissing the UI will lead to a `dismiss` state. It is up to each vendor (ads & analytics) how they would process `accept`, `reject` and `dismiss`.
 
-##### What is `checkConsentHref`? And why is it mandatory? 
+##### What is `checkConsentHref`? And why is it mandatory?
```
Comment (lannka) [inline extensions/amp-consent/amp-consent.md]: update this

Comment (zhouyx) [reply extensions/amp-consent/amp-consent.md]: Ha found we explained that it's not mandatory with `promptIfUnknownForGeoGroup` right below. https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/amp-consent.md#what-is-checkconsenthref-and-why-is-it-mandatory

File: extensions/amp-consent/amp-consent.md (hunk 1 above)
Comment (zhouyx) [reply extensions/amp-consent/amp-consent.md]: Changed to CORS HTTPS POST request....

**Pull request merged**